docs/landlock.md

```diff
@@ -29,18 +29,40 @@ Linux kernel confirms Landlock support with above message in dmesg.
 
 ## Implementation Details
 
-To enable Landlock, Cloud-Hypervisor process needs the full list of files it
-needs to access over its lifetime. Landlock is enabled in the `vm_create` stage.
+Landlock is enabled in `vm_create` stage of guest boot.
```

Comment:
The `Implementation Details` section is too short. Let's merge it to the beginning of the next section `Enable Landlock`. Also, I feel the original content is useful and should be kept , e.g. "To enable Landlock, Cloud-Hypervisor process needs the full list of files it needs to access over its lifetime. "

Reply:
Updated the doc. Thanks!